### PR TITLE
Fixed all internal broken links and fixed some typos

### PR DESCRIPTION
**Pull request by strider**

### CODE REVIEW

File: _includes/manuals/1.1/1.1_architecture.md [hunk 1]
```diff
@@ -11,6 +11,6 @@ Puppet CA.
 
 ## Smart-Proxy
 
-A [Smart-Proxy](#4.3SmartProxies) is located on or near a machine that performs a specific function and helps foreman orchestrate the process of commissioning a new host. Placing the proxy on or near to the actual service will also help reduce latencies in large distributed organizations.
+A [Smart-Proxy](manuals/1.1/index.html#4.3SmartProxies) is located on or near a machine that performs a specific function and helps foreman orchestrate the process of commissioning a new host. Placing the proxy on or near to the actual service will also help reduce latencies in large distributed organizations.
```
Comment (domcleal) [inline _includes/manuals/1.1/1.1_architecture.md:14]: It bothers me that we need to do this, but I can't think of an alternative, apart from removing the `<base>` tag that was added in 06889414 (@frimik), but that stops us being able to support non-/ base URLs.  Oh well.

File: _includes/manuals/1.1/1.2_release_notes.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ Upgrades from previous versions of Foreman are mostly automatic when using packa
 Here are some of the most prominent features of Foreman 1.1
 
 * Support for Puppet 3
-* Locations & organizations - http://projects.theforeman.org/projects/foreman/wiki/LocationsAndOrganizations
-* Parameterized classes - http://projects.theforeman.org/projects/foreman/wiki/Parameterized_class_support
-* V1 of a robust, versioned API - http://theforeman.org/api.html
+* Locations & organizations - [http://projects.theforeman.org/projects/foreman/wiki/LocationsAndOrganizations](http://projects.theforeman.org/projects/foreman/wiki/LocationsAndOrganizations)
```
Comment (domcleal) [inline _includes/manuals/1.1/1.2_release_notes.md]: Maybe just link the bullet point text straight to the URL?

File: _includes/manuals/1.1/4.1.1_authentication.md [hunk 3]
```diff
@@ -14,9 +14,9 @@ and restart Foreman
 
 #### Setting up
 
-Go to  More -> Users -> LDAP Authentication
+Go to **More -> Users -> LDAP Authentication**
```
Comment (domcleal) [inline _includes/manuals/1.1/4.1.1_authentication.md]: This is a different style to the one you used below, which was italics (one *) and no hyphen.  My preference is italics, but whichever, so long as it's consistent.

**Comment by domcleal** [comment]
That's great, thanks @strider!  Just a couple of comments above - once you've fixed them, squash into the previous commit (`git add`; `git commit --amend`) and then force-push the branch (`git push -f`).

**Comment by strider** [comment]
The commit a61e517 seems good for me.

**Comment by domcleal** [comment]
Squashed and merged as f0d287a.  Thanks for the contribution @strider!

**Pull request closed without merging**